media-sound/setbfree/setbfree-9999.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit toolchain-funcs multilib
+
+DESCRIPTION="MIDI controlled DSP tonewheel organ"
+HOMEPAGE="http://setbfree.org"
+
+if [ "${PV}" = "9999" ]; then
+	inherit subversion git-r3
+	EGIT_REPO_URI="https://github.com/pantherb/setBfree.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/pantherb/setBfree/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="convolution"
+
+RDEPEND="media-sound/jack-audio-connection-kit
+	>=media-libs/alsa-lib-1.0.0:=
+	media-libs/liblo:=
+	media-libs/lv2
```

Comment:
It's a mess of automagic junk in a Makefile with no easy way to configure what goes in and what doesn't. As far as I can tell, the UI (even in standalone mode) depends on LV2. You can build the synth core without LV2 and the UI, but there's no way to force that (well, unless you do something ugly like set FONTFILE to a non-existent file to force the UI test to fail). And the synth core by itself is of limited use unless you already have all the presets you need (headless mode?) or are happy using MIDI to control everything.